src/rulesets/asyncapi/functions/utils/isObject.ts

```diff
@@ -0,0 +1,5 @@
+import type { Dictionary } from '@stoplight/types';
+
+export function isObject(value: unknown): value is Dictionary<unknown> {
+  return value !== null && typeof value === 'object';
+}
```

Comment:
Should we also check if `!Array.isArray(value)`? Because `isObject([1,2])` would return `true` and array can't be a `Dictionary` type, right?

Also, is there some lodash equivalent we could use?